base/applications/cmdutils/label/lang/ro-RO.rc

```diff
@@ -1,20 +1,26 @@
-/* Translator: Ștefan Fulea (stefan dot fulea at mail dot com) */
+/*
+ * PROJECT:     ReactOS Label Command
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Romanian resource file
+ * TRANSLATORS: Copyright 2011-2019 Ștefan Fulea <stefan.fulea@mail.com>
+ *              Copyright 2023 Andrei Miloiu <miloiuandrei@gmail.com>
+ */
 
 LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
 
 STRINGTABLE
 BEGIN
 //    STRING_LABEL_HELP "Afișează sau modifică eticheta unei unități (de stocare).\n\nLABEL [unitate:][etichetă]\n"
-    STRING_LABEL_HELP "Creates, changes or deletes the volume label of a disk.\n\n\
-LABEL [drive:][label]\n\n\
-  drive:          Specifies the drive letter of a drive.\n\
-  label           Specifies the label of the drive."
-    STRING_LABEL_TEXT1 "Volumul unității %c: este etichetat ca %s\n"
-    STRING_LABEL_TEXT2 "Volumul unității %c: nu este etichetat.\n"
+    STRING_LABEL_HELP "Crează, schimbă sau șterge denumirea volumului a unui disc.\n\n\
+LABEL [disc:][denumire]\n\n\
+  disc:           Specifică litera unui disc.\n\
+  denumire        Specifică denumirea unui disc."
```

Comment:
"Disc", although may still be still encountered around in ReactOS resources, should be considered an obsolete term by now. The current information storage may not contain any discs nowadays. I'd translate the English "drive" as "dispozitiv" or "dispozitiv de stocare" in Romanian.

Reply:
I have a question: shouldn't we still keep it translated as "disc", in order to avoid any confusion with the translion of some words, like "hardware" and "device"? I'm okay if I change everything back.

Reply:
As I asked you before: shouldn't we stick with the word "disc", especially to avoid any confusion? I still think it's a better term.

Reply:
> I have a question: shouldn't we still keep it translated as "disc", in order to avoid any confusion with the translation of some words, like "hardware" and "device"?

I haven't thought about this too much myself before, and about this I admit to have just translated the English terms, obsoleted as they may have been. I can't say I understand why "disc" is a good choice, but I don't necessarily have to.

Reply:
So, do you mean to re-revert the word "disc"? I don't have time to change the translation everywhere.

Reply:
I don't understand anything.

Reply:
> So, do you mean to re-revert the word "disc"?
I don't understand anything.

I admitted that I didn't cared enough when it was me managing the translations. It was kind of messy before and it's good that you noticed.

> I don't have time to change the translation everywhere.

It's not necessary for everything to be taken care of now. You may note somewhere down to change it in the future, if you care enough.

Reply:
Ok. I re-reverted the change. I hope it's good now.